tuning/trackers/aimstack_tracker.py

```diff
@@ -97,7 +97,8 @@ def __init__(self, tracker_config: AimConfig):
             information about the repo or the server and port where aim db is present.
         """
         super().__init__(name="aim", tracker_config=tracker_config)
-        self.logger = logging.get_logger("aimstack_tracker")
+        # Get logger with root log level
+        self.logger = logging.getLogger()
```

Comment:
@Abhishek-TAMU any particular reason to remove module name from the `logger` I had kept it to segregate logs from various subcomponents.

Reply:
Same comment for `file_logging_tracker`

Reply:
Explained in the comment above..no need for futher explanation.

Reply:
Cool. Thank you!

Reply:
I still dont quite understand this. You're setting log_level at runtime to restrict the amount of logs that a user wants to see (`debug` vs `info` etc.). But if you don't do `logging.get_logger("aimstack_tracker")` and so on in each file, all the logs will say they come from `root`, not module specific, helpful but not as helpful if the module is also included.
Maybe it'd be clearer if you could share an example run of what the logs look like if user sets `debug` or `error` for example and we can see what the logs look like.